src/components/Pagination/Pagination.tsx

```diff
@@ -163,10 +167,17 @@ export function Pagination({
       />
     ));
 
+  const labelMarkup = label ? (
+    <div className={styles.Label} aria-live="polite">
+      <TextStyle variation="subdued">{label}</TextStyle>
```

Comment:
Should we have it always subdued, of maybe just subdue if `hasNext` and `hasPrevious` are both false 🤷‍♂ 

cc: @sarahill

Reply:
I think that is a good idea, makes sense for the indicator to also be subdued if the actions surrounding it are as well and vice versa 👍

Reply:
## Label with previous and next
![image](https://user-images.githubusercontent.com/42747959/65155153-ba2be100-d9fa-11e9-898b-d3f12fbf9e8a.png)

## Label without previous and next
![image](https://user-images.githubusercontent.com/42747959/65155223-e0518100-d9fa-11e9-96b1-3ec02676b5f4.png)